pkg/smokescreen/acl/v1/policy.go

```diff
@@ -0,0 +1,43 @@
+package acl
+
+import "fmt"
+
+// DecisionResult represents the result of checking the ACL
+type DecisionResult int
+
+const (
+	Allow DecisionResult = iota
+	AllowAndReport
+	Deny
+)
+
+func (d DecisionResult) String() string {
+	return [...]string{"Unknown", "Allow", "AllowAndReport", "Deny"}[d]
```

Comment:
I'm not well-versed in this notation but it looks like we may be missing an `Unknown` value for the `DecisionResult` enum

Reply:
You are 100% right. Fixing this actually uncovered another bug. I added some regression tests.